packages/serverpod/lib/src/server/server.dart

```diff
@@ -65,6 +65,13 @@ class Server {
   /// Central message dispatch for real time messages.
   MessageCentral messageCentral = MessageCentral();
 
+  /// Http headers used by all API responses. Defaults to allowing any
+  /// cross origin resource sharing (CORS).
+  final Map<String, dynamic> httpResponseHeaders;
+
+  /// Http headers used by OPTIONS responses.
```

Comment:
Documentation should either make clear that these are in addition to the `httpResponseHeaders` or (better) by default include all headers.